.circleci/config.yml

```diff
@@ -51,6 +73,10 @@ workflows:
           filters:
             branches:
               ignore: master
+      - validate_changed_seed:
```

Comment:
Hi @sandertan Does this take a long time to run?  If so, just a suggestion - instead of running it on every PR can it run on PRs with a label that indicates the seed database has changed?

Reply:
I think after pulling the Python Docker image, one of the first checks is whether the seed database file is changed. If it's not changed, the test will stop. I think it takes a minute to complete in that case.

Do you think that's reasonable on every PR? Else we can think of moving this check up.

Reply:
I think it makes sense to avoid pulling the docker image if it is not going to be used.

Reply:
@n1zea144 @pieterlukasse I've created a new branch and new PR with an update. This contains your suggestion. 
https://github.com/cBioPortal/datahub/pull/448

By pushing the branch to cBioPortal/datahub instead of TheHyve/datahub I bypassed the issue that I can't push datahub branches to non-cBioPortal remotes.